api/src/main/java/org/motechproject/nms/api/web/BaseController.java

```diff
@@ -46,6 +46,7 @@
     public static final String NOT_FOUND = "<%s: Not Found>";
     public static final String NOT_AUTHORIZED = "<%s: Not Authorized>";
     public static final String NOT_DEPLOYED = "<%s: Not Deployed In State>";
+    public static final String INVALID_TYPE = "<%s: Invalid Type>";
```

Comment:
Was this agreed to with IMI? Why introduce this as opposed to  not authorized? IMI tends to have a direct dependency on these return types.